tests/JMS/Serializer/Tests/Serializer/DeserializeExceptionTest.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/*
+ * Copyright 2015 Ivan Borzenkov <ivan.borzenkov@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace JMS\Serializer\Tests\Serializer;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use JMS\Serializer\Construction\UnserializeObjectConstructor;
+use JMS\Serializer\Context;
+use JMS\Serializer\Exception\RuntimeException;
+use JMS\Serializer\GraphNavigator;
+use JMS\Serializer\Handler\ArrayCollectionHandler;
+use JMS\Serializer\Handler\ConstraintViolationHandler;
+use JMS\Serializer\Handler\DateHandler;
+use JMS\Serializer\Handler\FormErrorHandler;
+use JMS\Serializer\Handler\HandlerRegistry;
+use JMS\Serializer\Handler\PhpCollectionHandler;
+use JMS\Serializer\JsonDeserializationVisitor;
+use JMS\Serializer\JsonSerializationVisitor;
+use JMS\Serializer\Metadata\Driver\AnnotationDriver;
+use JMS\Serializer\Naming\CamelCaseNamingStrategy;
+use JMS\Serializer\Naming\SerializedNameAnnotationStrategy;
+use JMS\Serializer\Serializer;
+use JMS\Serializer\Tests\Fixtures\AuthorList;
+use JMS\Serializer\VisitorInterface;
+use Metadata\MetadataFactory;
+use PhpCollection\Map;
+use Symfony\Component\Translation\IdentityTranslator;
+use Symfony\Component\Translation\MessageSelector;
+
+class DeserializeExceptionTest extends \PHPUnit_Framework_TestCase
```

Comment:
why not add this tests in `BaseSerializationTest` to avoid all that stuff in `setUp` ?